src/microbe_stage/Patch.cs

```diff
@@ -41,7 +41,7 @@ public class Patch
     ///   The current effects on patch node (shown in the patch map)
     /// </summary>
     [JsonProperty]
-    private readonly List<WorldEffectVisuals> activeWorldEffectVisuals = new();
+    private readonly List<WorldEffectVisuals.WorldEffectTypes> activeWorldEffectVisuals = new();
```

Comment:
Seeing as this nested enum is used all over, I think it would be better to not have the enum be nested....

Though I'm not sure why there needs to be a Visuals enum and then a type enum inside that. To me that seems the opposite, the visual (as seen on line 605) seems to still be the primary data model...